scripts/spec/kickstart_generator_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper'
+require 'kickstart_generator'
+
+describe Build::KickstartGenerator do
+  describe "#run" do
+    def build_base
+      data_file_path("kickstart_generator")
```

Comment:
For what?  I'm using the directory structure that would be in place during a real build run which is recreated in the `data/kickstart_generator` directory.  In this tree are the base.ks.erb and partials I'm using to test; assuming I'm not also going to generate those files from within the test I can't be creating the build base here as a temp directory.